formatter/index.js

```diff
@@ -0,0 +1,7 @@
+import { format_raw } from "@ioannisnezis/sparql-language-server"
+
+const formatButton = document.getElementById("formatButton");
+formatButton.addEventListener('click', () => {
+
+	editor.setValue(format_raw(editor.getValue()));
+})
```

Comment:
Is it possible to also re-export the WASM function as JS function and attach the listener in `qleverUI.js`?

Reply:
Yes, webpack can be configured to expose exported functions.
I'll push a commit addressing this.

Reply:
Nice. Afaik an ES module would be the way to go nowadays. But this seems to break other parts of the JS codebase for me... But that's not something to fix in this PR.